container/apbf/generateapbftable.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+//+build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+// calcRateKey is used as the key when caching results for the false positive
+// and query access calculations.
+type calcRateKey struct {
```

Comment:
Why isn't `calcFPRateKey` reused here?

Reply:
Because it's in a different package and not accessible.  This is code that is run from the `main` package during for `go generate`.